text/0000-project-ffi-unwind.md

```diff
@@ -0,0 +1,211 @@
+- Feature Name: `project-unwind-FFI`
+- Start Date: 2019-10-27
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+* To create a "project group" with the purpose of designing subsequent RFCs to
+  flesh out the details of the "C unwind" ABI
```

Comment:
```suggestion
  flesh out the details of the "C unwind" ABI or equivalent language feature
```

Reply:
Ah, good catch. Better yet, here's the top-level summary of our charter:

```suggestion
  extend the language to support unwinding that crosses FFI boundaries
```